doc/news/_preparation_next_release.md

```diff
@@ -105,7 +105,16 @@ The text below summarizes updates to the [C (and C++)-based libraries](https://w
 Bindings allow you to utilize Elektra using [various programming languages](https://www.libelektra.org/bindings/readme). This section keeps
 you up to date with the multi-language support provided by Elektra.
 
-### <<Binding1>>
+### Java binding
+
+- Renamed zero argument static factory method `Key::createNameless` to `Key::create`.
+- Removed signatures publishing the error key based error handling approach to the Java binding consumer. Such arguments are now created implicitly by the binding. In case of an exceptional state, appropriate exceptions are thrown. Such exceptions provide access to the underlying key containing warning and error information as meta data. Affected signatures:
```

Comment:
Please write here what API users of these functions would do now.

Reply:
done